pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerImpl.java

```diff
@@ -1442,11 +1454,13 @@ public void seek(long timestamp) throws PulsarClientException {
     public CompletableFuture<Void> seekAsync(long timestamp) {
         if (getState() == State.Closing || getState() == State.Closed) {
             return FutureUtil
-                    .failedFuture(new PulsarClientException.AlreadyClosedException("Consumer was already closed"));
+                .failedFuture(new PulsarClientException.AlreadyClosedException(
+                    String.format("[%s][%s] Consumer was already closed", topicName.toString(), subscription)));
```

Comment:
Same comments as below.

"Consumer %s was already closed when seeking subscription %s of topic %s to timestamp %d", consumerName, subscription, topicName, timestamp